demisto_sdk/commands/json_to_outputs/tests/json_to_outputs_test.py

```diff
@@ -102,14 +105,51 @@ def test_json_to_outputs__a_list_of_dict():
     yaml_output = parse_json(
         data='[{"a": "b", "c": "d"}, {"a": 1}]',
         command_name='jira-ticket',
-        prefix='Jira.Ticket'
+        prefix='Jira.Ticket',
+        description_dictionary={"a": DUMMY_FIELD_DESCRIPTION}
     )
 
-    assert yaml_output == '''arguments: []
+    assert yaml_output == f'''arguments: []
 name: jira-ticket
 outputs:
 - contextPath: Jira.Ticket.a
+  description: {DUMMY_FIELD_DESCRIPTION}
+  type: Number
+- contextPath: Jira.Ticket.c
   description: ''
+  type: String
+'''
+
+
+@pytest.mark.parametrize('description_dictionary, expected_a_description',
+                         [
+                             ({}, "''"),
+                             ({'nonexistent_field': 'foo'}, "''"),
+                             (None, "''"),
+                             ("", "''"),
+                             ({"q": "this should not show as q is not in the data"}, "''")
+                         ])
+def test_json_to_outputs__empty_description_dictionary(description_dictionary, expected_a_description):
+    """
+    Given
+        - A list of dictionaries
+    When
+        - Passed to json_to_outputs
+    Then
+        - ensure the returned type is correct
+    """
```

Comment:
Since you're using parametrize, better change the name of the test.
Add a test for file path as well, and better add a test mocking `input_multiline` to test input flow.

As a side note, I understand that the standard of this page is to test `json_to_outputs` via `parse_json`, but the naming is confusing.

Reply:
changed to `test_json_to_outputs__invalid_description_dictionary`, is that okay?

I suppose the confusing name comes from the file name rather than the actual command (with the same name)

Reply:
Yeah, `invalid` is a good option